example/android/app/build.gradle

```diff
@@ -93,6 +96,15 @@ def enableSeparateBuildPerCPUArchitecture = false
  */
 def enableProguardInReleaseBuilds = false
 
+/**
+ * Use international variant JavaScriptCore
+ * International variant includes ICU i18n library and necessary data allowing to use
```

Comment:
... includes _the_ ICU ...
... allowing ~to~ use _of methods_ ...

Reply:
This is an autogenerated file